packages/integration/src/bitbucketServer/config.ts

```diff
@@ -46,6 +53,11 @@ export type BitbucketServerIntegrationConfig = {
    * If no token is specified, anonymous access is used.
    */
   token?: string;
+
+  /**
+   * Custom headers to use for request to a Bitbucket Server provider.
+   */
+  headers?: { [key: string]: string };
```

Comment:
Should this be HeadersKV too? Or do you want to just drop `HeadersKV`?

Reply:
seems like you could just use `Record<string, string>` as type I think.

Or maybe you want to support multiple strings like `Record<string, string[]>` or `Record<string, string | string[]>`.